src/30_Advanced/Video_Carousels_with_amp-carousel.html

```diff
@@ -0,0 +1,115 @@
+  <script async custom-element="amp-vimeo" src="https://cdn.ampproject.org/v0/amp-vimeo-0.1.js"></script>
+  <script async custom-element="amp-dailymotion" src="https://cdn.ampproject.org/v0/amp-dailymotion-0.1.js"></script>
+  <script async custom-element="amp-twitter" src="https://cdn.ampproject.org/v0/amp-twitter-0.1.js"></script>
+  <script async custom-element="amp-facebook" src="https://cdn.ampproject.org/v0/amp-facebook-0.1.js"></script>
+  <script async custom-element="amp-brightcove" src="https://cdn.ampproject.org/v0/amp-brightcove-0.1.js"></script>  
+  <script async custom-element="amp-iframe" src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js"></script>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    amp-carousel {
+      background: black;
+    }
+    .container {
+      display: flex;
+      flex-flow: column;
+      justify-content: center;
+      height: 100%;
+    }
+    .child {
+      width: 100%;
+      height: 55%;
+    }
+
+
+</style>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+    <!-- #### A Video Carousel embedded in an amp-carousel component -->
+  <!-- Display a list of videos as slides. For more info about `amp-carousel` take a look [here](https://ampbyexample.com/components/amp-carousel/). -->
+  <amp-carousel width="450"
+      height="450"
+      layout="responsive" type="slides">
+    <amp-video width="480" height="270" src="/video/tokyo.mp4"
+               poster="/img/tokyo.jpg"
+               layout="responsive"
+               controls>
+        <div fallback>
+        <p>Your browser doesn't support HTML5 video.</p>
+        </div>
+        </amp-video>
+    <amp-brightcove
+                  data-account="906043040001"
+                  data-video-id="1401169490001"
+                  data-player-id="180a5658-8be8-4f33-8eba-d562ab41b40c"
+                  layout="responsive" width="400" height="300">
+    </amp-brightcove>
+    <amp-dailymotion
+      data-videoid="x2m8jpp"
+      layout="responsive"
+      data-ui-highlight="FF4081"
+      width="480" height="270">
+    </amp-dailymotion>
+    <div>
+      <div class="container">
```

Comment:
I added a comment about this in the "The Video Carousel" section. Adding a comment in the middle of the amp-carousel element was causing the amp-carousel tag to be closed and the remaining videos were removed from the carousel. Any idea how to achieve this?